test_requirements.pip

```diff
@@ -6,3 +6,5 @@ Pillow==4.2.1
 pandas==0.19.1
 scikit-learn==0.19.1
 h5py==2.7.1
+lightgbm~=2.1.0
```

Comment:
Do you mean `==` not `~=`?

Reply:
No, I meant `~=`. Anything `2.1.x`. But we can change it if that's not a format you guys like.